apps/src/codebridge/FileBrowser/hooks/useFileUploader.tsx

```diff
@@ -1,144 +1,41 @@
 import {sendCodebridgeAnalyticsEvent} from '@codebridge/utils/analyticsReporterHelper';
```

Comment:
github really made this file change unclear.

The original file is moved to `lab2/hooks` with some tweaks around how the analytics events are sent.

_This_ file now just wraps the generic one and builds the `sendAnalyticsEvent` callback to hand through to the generic file uploader.